### PR TITLE
Updated settings to only include lists and '

### PR DESCRIPTION
**Pull request by schacki**
Substituted " with ' in files
Migrated tuples  to lists

### CODE REVIEW

File: {{cookiecutter.project_slug}}/config/settings/local.py [hunk 1]
```diff
@@ -50,14 +50,14 @@
 
 # django-debug-toolbar
 # ------------------------------------------------------------------------------
-MIDDLEWARE += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
-INSTALLED_APPS += ('debug_toolbar', )
+MIDDLEWARE += ['debug_toolbar.middleware.DebugToolbarMiddleware']
```
Comment (pydanny) [inline {{cookiecutter.project_slug}}/config/settings/local.py]: No trailing comma?

**Comment by pydanny** [comment]
No trailing comma?

**Comment by pydanny** [comment]
... and thanks for this pull request. 😄

**Comment by Sgiath** [comment]
And what about other files? For example [here](https://github.com/pydanny/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/config/wsgi.py#L28)

**Comment by schacki** [comment]
Sorry for the late reply. Trailing commas are not used consistently, I am happy to make it consistent in settings files. What do you think? Is there any place in the docs, where to put this as a requirement for contributors? The same for the "?

**Comment by schacki** [comment]
So what is the preference, with or without trailing comma?

**Comment by webyneter** [comment]
@schacki, trailing comma is always a great, defensive choice, IMO.

**Pull request merged**